ga4gh/datarepo.py

```diff
@@ -104,7 +104,12 @@ def getOntology(self, name):
         """
         Returns ontologies
         """
-        return self._ontologyNameMap[name]
+        ret = {}
+        try:
```

Comment:
not that both of these approaches create an dict object that is then discarded even when the name map contains the object.